app/code/community/Bolt/Boltpay/Model/BoltOrder.php

```diff
@@ -345,6 +351,80 @@ protected function getCorrectedTotal($projectedTotal, $magentoDerivedCartData)
         return $magentoDerivedCartData;
     }
 
+    /**
+     * Checks if billing address of a quote has all the expected fields.
+     * If not, the address is updated by using the provided shipping address.
+     *
+     * @param Mage_Sales_Model_Quote $quote Quote that is expected to contain both
+     *                                      a billing and shipping address.
+     *
+     * @return bool  true if a correction is made, otherwise false
```

Comment:
how about "true if billing address was updated"?

Reply:
seems the same info to me.  I'll change as you have requested.

Reply:
Actually, on second thought, `update` to me implies a change to the database while corrected does not necessarily give this connotation.  Corrected seems like a better choice for this reason.